Backend/tests/test_controllerResponse.py

```diff
@@ -0,0 +1,56 @@
+import unittest
+import sys
+from os import path
+sys.path.append( path.dirname( path.dirname( path.abspath(__file__) ) ) )
+from utils.ResponseCreation import *
+
+
+class ControllerResponseTests(unittest.TestCase):
+  @classmethod
+  def setUpClass(self):
+    self.response = ControllerResponse()
```

Comment:
Why do you do this if in every test method you have response = ControllerResponse()? Consider using setUp() which does something at the start of every test case